backend/app/api/endpoints/knowledge.py

```
@@ -740,6 +745,222 @@ def get_table_document(
     return KnowledgeDocumentResponse.model_validate(document)
 
 
+@table_router.post("/validate-url", response_model=TableUrlValidationResponse)
+async def validate_table_url(
+    data: TableUrlValidationRequest,
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)